networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -165,6 +165,11 @@ def _get_uplinksets_from_lig(self, network_type, lig_list):
         return uplinksets
 
     def _create_network_on_oneview(self, name, network_type, seg_id):
+        compatible_networks = (
+            self.oneview_client.ethernet_networks.get_by('name', name))
+        if compatible_networks:
```

Comment:
should we check at least if it has the same segmentation id and network type?

Reply:
Not, because neutron make rowback when network creation fails, and hpOneView throws a exception.

Reply:
Reinaldo think we should check all parameters... an update on VLAN ID at neutron (which is not still mapped into OneView) will yield to inconsistency.